scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -5,23 +5,49 @@ set -e
 # show commands
 set -x
 
+ARCH=$(uname -m)
 export CI="openshift"
-make configure-installer-tests-cluster
+if [ "${ARCH}" == "s390x" ]; then
+    make configure-installer-tests-cluster-s390x
+else
+    make configure-installer-tests-cluster
+fi
 make bin
 mkdir -p $GOPATH/bin
 go get -u github.com/onsi/ginkgo/ginkgo
 export PATH="$PATH:$(pwd):$GOPATH/bin"
 export ARTIFACTS_DIR="/tmp/artifacts"
 export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
 
-# Integration tests
-make test-integration
-make test-integration-devfile
-make test-cmd-login-logout
-make test-cmd-project
-make test-operator-hub
+# import the odo-init-image for s390x arch
+if [ "${ARCH}" == "s390x" ]; then
+    export ODO_BOOTSTRAPPER_IMAGE=quay.io/zhengxiaomei123/odo4z:1.1.3-s390x
```

Comment:
is this reasonable change here?

Reply:
> is this reasonable change here?

We reset odo-init-image to make it support s390x arch. Once the odo-init-image released by redhat support multi-arch, the step will be removed.

Reply:
Multi-arch support has been added through https://github.com/openshift/odo-init-image/pull/67

Reply:
> Multi-arch support has been added through [openshift/odo-init-image#67](https://github.com/openshift/odo-init-image/pull/67)

Yes,  it does not release the multi-arch odo-init-image to registry.access.redhat.com/openshiftdo/odo-init-image-rhel7 which is hardcode in odo `defaultBootstrapperImage` . So We reset the value here, when the `defaultBootstrapperImage` support multi-arch, I will remove it.

Reply:
We are currently having configuration issues in internal build system. The image was released, but is only available on `registry.redhat.io` (https://catalog.redhat.com/software/containers/ocp-tools-4/odo-init-container-rhel8/5ee72e01e1600e9787833e8b?container-tabs=gti). You can see it is available for multiple archs 

Once that configuration is fixed, we will re-push that image, to registry.access.redhat.com with the same tag. Once this happens, we should be able to have one unified image.